sdk/python/kfp/cli/diagnose_me/kubernetes_cluster.py

```diff
@@ -0,0 +1,122 @@
+# Lint as: python3
+# Copyright 2019 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License,Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Functions for collecting diagnostic information on Kubernetes cluster."""
+
+import enum
+from typing import List, Text, Optional
+from . import utility
+
+
+class Commands(enum.Enum):
+  """Enum for kubernetes commands."""
+  K8_CONFIGURED_CONTEXT = 1
+  PODS = 2
+  PVCS = 3
+  PVS = 4
+  SECRETS = 5
+  SERVICES = 6
+  KUBECTL_VERSION = 7
+
+
+_command_string = {
+    Commands.K8_CONFIGURED_CONTEXT: 'config view',
+    Commands.PODS: 'get pods',
```

Comment:
Nit: "Command" implies an action, a verb, but "PODS" is a noun.

Reply:
This is a pretty good point it bothered me as well. I changed the naming a couple of times before settling on this. changing the command to command_phrase, command_string ect or changing the actual commands enums to get_pods makes the calls much longer though I did not think it added much benefit. 

Let me know if keeping as is would be OK, otherwise i'm open to changing it.

Reply:
You can keep it, but later you might see the need to do so anyways.
P.S. When I see `Commands.PODS` somewhere in the code, I do not really know what it does.
With `Commands.GET_PODS` I know.

Reply:
agreed, updated accordingly. Also applied to command names in other files as well.